M/msquic/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "msquic"
+version = v"1.5.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/microsoft/msquic.git",
+              "a2c19d3ac386b8bb923e14d2607d42bffcfca0c7"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/msquic
+git submodule update --init --recursive
+mkdir build && cd build
```

Comment:
The finding about `static_assert` is very useful, I could go on a little bit with this:
```suggestion
mkdir build && cd build
export CFLAGS="-Dstatic_assert=_Static_assert"
```